### PR TITLE
[ADD] Employees: adding overview doc larm

### PR DESCRIPTION
**Pull request by larm-odoo**
Created a new Employees app section under HR and added the first new doc to this, the overview.

### CODE REVIEW

File: content/applications/hr/employees.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+:nosearch:
+:show-content:
+:hide-page-toc:
+:show-toc:
```
Comment (jcs-odoo) [inline content/applications/hr/employees.rst:4]: If you move your content to this page, no need for any of these options now.
```suggestion
```

However, if you add sub-pages, you'll need this:
```suggestion
:show-content:
:show-toc:
```

File: content/applications/hr/employees/employees_overview.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+.. image:: employees_overview/report-contracts.png
+   :align: center
+   :alt: View of all contracts in their various stages.
+
+Employee Presence
+-----------------
+
+The :guilabel:`Employee Presence` report shows how many employees are currently working or absent,
+and the options to approve :guilabel:`Time Off Requests`, or contact the employee via text  or email
+regarding their attendance can be done here.
```
Comment (hojo-odoo) [inline content/applications/hr/employees/employees_overview.rst:78]: Rephrase
```suggestion
The :guilabel:`Employee Presence` report shows how many employees are currently working or absent,
along with the options for managing :guilabel:`Time Off Requests`. Information on how to contact the employee via text or email is also located here.
```

File: content/applications/hr/employees/employees_overview.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+   :alt: View of all employees who are absent and present.
+
+Configuration
+=============
+
+The :guilabel:`Configuration` section is where general employee settings are configured, and
+job positions and departments are created and managed. Create onboarding and offboarding plans in
+detail, as well as create and manage contract templates, to more easily handle new recruits. Gamify
+aspects of an employee's job through the creation of challenges and badges, and view all past goals,
+with metrics achieved.
```
Comment (hojo-odoo) [inline content/applications/hr/employees/employees_overview.rst:91]: Word choice
```suggestion
including metrics achieved.
```

File: content/applications/hr/employees/employees_overview.rst [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+.. image:: employees_overview/employee_presence.png
+   :align: center
+   :alt: View of all employees who are absent and present.
+
+Configuration
+=============
+
+The :guilabel:`Configuration` section is where general employee settings are configured, and
+job positions and departments are created and managed. Create onboarding and offboarding plans in
+detail, as well as create and manage contract templates, to more easily handle new recruits. Gamify
```
Comment (hojo-odoo) [inline content/applications/hr/employees/employees_overview.rst:89]: Rephrase
```suggestion
job positions and departments are created and managed. From here, it is possible to create onboarding and offboarding plans in detail, as well as create and manage contract templates to more easily handle new recruits.
```

**Review by hojo-odoo — CHANGES_REQUESTED** [review]

**Comment by larm-odoo** [comment]
This overview was written when I first started writing the HR docs. There were no docs, so I started with these overviews. In the end, this was not the right path to take. Now, I have many HR docs written and published- all very thorough and detailed, so these are no longer needed. Closing this PR, along with the other 3 original "overview:" HR docs created at the same time.

**Pull request closed without merging**